to_json.wdl

```diff
@@ -0,0 +1,237 @@
+version 1.0
+
+workflow to_json {
+    input {
+        File gff
+		File readcount
+		String prefix
+    }
+
+    call rctojson{
+        input:
+        gff = gff,
+        readcount = readcount,
+        prefix = prefix
+    }
+}
+
+task rctojson{
+	input {
+		File gff
+		File readcount
+		String prefix
+		String container = "microbiomedata/meta_t@sha256:7e2f1566d3aa64ad55981c5e294b144b20889cbc6d6d24e54d364f379c782324"
+	}
+	command <<<
+		python <<CODE
+		# Imports #######################################################################################################################################
+		import os
+		import json
+		import pandas as pd
+		import gffutils
+		# Definitions #####################################################################################################################################
+		# Functions #######################################################################################################################################
+		def final_jsons(gff_in = "test_data/paired.gff", rc_in = "test_data/paired.rc", 
+						gff_json = "paired.gff.json", 
+						rc_json = "paired.rc.json",
+						gff_rc_json = "gff_rc.json",
+						cds_json = "cds_counts.json",
+						sense_json = "sense_counts.json",
+						anti_json = "antisense_counts.json",
+						sorted_json = "sorted_features.json",
+						sorted_tsv = "sorted_features.tsv",
+						top100_json = "top100_features.json",
+						prefix = ""
+						):
+			"""
+			Generate JSON files for NMDC EDGE MetaT output tables. 
+			Combine JSON files from GFF and read count TSV using pandas
+			"""
+			if (prefix != ""):
+				gff_json = prefix + "_paired.gff.json"
+				rc_json = prefix + "_paired.rc.json"
+				gff_rc_json = prefix + "_gff_rc.json"
+				cds_json = prefix + "_cds_counts.json"
+				sense_json = prefix + "_sense_counts.json"
+				anti_json = prefix + "_antisense_counts.json"
+				sorted_json = prefix + "_sorted_features.json"
+				sorted_tsv = prefix + "_sorted_features.tsv"
+				top100_json = prefix + "_top100_features.json"
+			
+			gff_obj = GTFtoJSON(gff_in, gff_json).gtf_json()
+			
+			rc_obj = TSVtoJSON(rc_in, rc_json).tsv_json()
+
+			gff_pd = pd.read_json(gff_json)
+			rc_pd = pd.read_json(rc_json)
+			gff_rc_pd = pd.merge(gff_pd, rc_pd, on = ["id", "seqid", "featuretype", "strand", "length"])
+
+			cds_only = gff_rc_pd[gff_rc_pd['featuretype'] == "CDS"]
+
+			sense_reads = cds_only[cds_only['strand'] == "+"].drop(columns = ["antisense_read_count", 
+												"meanA",
+												"medianA",
+												"stdevA"])
+			antisense_reads = cds_only[cds_only['strand'] == "-"].drop(columns = ["sense_read_count", 
+												"mean",
+												"median",
+												"stdev"])
+
+			sorted_features = gff_rc_pd.sort_values(by='sense_read_count', ascending=False)
+			top100 = sorted_features[:100]
+			
+			write_json(gff_rc_pd.to_dict(orient="records"), gff_rc_json)
```

Comment:
You could use https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_json.html